expect.ts

```diff
@@ -1,4 +1,4 @@
-import type { Locator } from "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/refs/heads/master/types/k6/browser/index.d.ts";
+import type { Locator } from "k6/browser";
```

Comment:
The URLs weren't working for me so I replaced them with the types from `@types/k6` instead.